dev/dune

```diff
@@ -4,7 +4,10 @@
  (synopsis "Coq's Debug Printers")
  (wrapped false)
  (modules top_printers vm_printers)
- (libraries coq-core.toplevel coq-core.plugins.ltac))
+ ; NB currently we have no "install_printer" using ltac2,
+ ; but we still want to load_printer ltac2
+ ; so that its genarg printers are present in the debugger's printer process
+ (libraries coq-core.vernac coq-core.plugins.ltac coq-core.plugins.ltac2))
```

Comment:
this seems to cause double loading of cc_core in coqtop.byte
once statically linked because coqtop.byte depends on this for Drop
then a second time through dynlink when loading the prelude